tests/app/notifications/test_validators.py

```diff
@@ -55,135 +56,66 @@ def enable_redis(notify_api):
         yield
 
 
-@pytest.mark.parametrize('key_type', ['test', 'team', 'normal'])
-def test_check_service_message_limit_in_cache_with_unrestricted_service_is_allowed(
-        key_type,
-        sample_service,
-        mocker):
-    mocker.patch('app.notifications.validators.redis_store.get', side_effect=[
-        None,  # The serialised service
-        1,  # The rolling count
-    ])
-    mocker.patch('app.notifications.validators.redis_store.set')
-    mocker.patch('app.notifications.validators.services_dao')
-    serialised_service = SerialisedService.from_id(sample_service.id)
-
-    check_service_over_daily_message_limit(key_type, serialised_service)
-    app.notifications.validators.redis_store.set.assert_called_once_with(
-        f'service-{serialised_service.id}',
-        ANY,
-        ex=ANY,
-    )
-    assert not app.notifications.validators.services_dao.mock_calls
-
-
-@pytest.mark.parametrize('key_type', ['test', 'team', 'normal'])
+@pytest.mark.parametrize('key_type', ['team', 'normal'])
 def test_check_service_message_limit_in_cache_under_message_limit_passes(
```

Comment:
Slightly out-of-scope: can we fix the test function names? Normally the first part is the name of the function under test, which in this case is `check_service_over_daily_message_limit`.

Later on, some of the test functions are missing the name altogether e.g. `test_should_not_interact_with_cache_for_test_key`. Can we fix those too?

Reply:
I've fixed all the function names